docuware-connector-product/README.md

```diff
@@ -0,0 +1,45 @@
+# DocuSign Connector
```

Comment:
Hi @andreasbalsiger 
The DocuWare connector is technically ready to enroll into the Axonivy Market.
Any wishes for the public README, that praises the product?